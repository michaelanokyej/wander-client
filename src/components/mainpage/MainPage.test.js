import React from "react";
import MainPage from "./MainPage"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`MainPage component`, () => {
    it(`renders the MainPage component`, () => {
        const wrapper = shallow(<MainPage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});