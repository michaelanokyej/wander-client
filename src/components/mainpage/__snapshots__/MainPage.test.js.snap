// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage component renders the MainPage component 1`] = `
<main
  className="App__main"
>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/home"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/users/:userId"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/signup"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/post-a-tour"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/login"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/tours/:tourId"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/search/tours"
  />
</main>
`;
